# DP: #212912
# DP: on alpha-linux, make -mieee default and add -mieee-disable switch
# DP: to turn default off (doc patch)

---
 gcc/doc/invoke.texi |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

--- a/src/gcc/doc/invoke.texi
+++ b/src/gcc/doc/invoke.texi
@@ -9980,6 +9980,13 @@ able to correctly support denormalized numbers and exceptional IEEE
 values such as not-a-number and plus/minus infinity.  Other Alpha
 compilers call this option @option{-ieee_with_no_inexact}.
 
+DEBIAN SPECIFIC: This option is on by default for alpha-linux-gnu, unless
+@option{-ffinite-math-only} (which is part of the @option{-ffast-math}
+set) is specified, because the software functions in the GNU libc math
+libraries generate denormalized numbers, NaNs, and infs (all of which
+will cause a programs to SIGFPE when it attempts to use the results without
+@option{-mieee}).
+
 @item -mieee-with-inexact
 @opindex mieee-with-inexact
 This is like @option{-mieee} except the generated code also maintains
